app/metrics/templates/metrics/tess-import.html

```diff
@@ -0,0 +1,181 @@
+{% extends "common/base.html" %}
+{% block content %}
+    <h1>{{ title }}</h1>
+    {% include 'common/tabs.html' %}
+    {% include 'metrics/data-warning.html' %}
+    <link rel="stylesheet" property="stylesheet" href="https://elixirtess.github.io/TeSS_widgets/css/tess-widget.css"/>
```

Comment:
While I think this make sense from a code sharing perspective this will make it harder to set up a truly local development environment. Is `https://elixirtess.github.io/TeSS_widgets/css/tess-widget.css` and `https://elixirtess.github.io/TeSS_widgets/js/tess-widget-standalone.js` and something that changes often? If it they are not then I would suggest we add copies of these to  the repo, unless you think that is a bad idea for some other reason.